### DISCUSSION TITLE
Παραδοτέο 2 - Π2019061

### DISCUSSION THREAD

**Discussion by tilav** [courses-ionio/sw#1259] [Show and tell]
Για το παραδοτέο της 2ης εβδομάδας υλοποίησα την άσκηση γραμμής εντολών: "Send notifications to your desktop-mobile".

[Asciinema](https://asciinema.org/a/h9udVZ5zDZvbfdjGuWa4OOCuh)
[Η αναφορά μου](https://github.com/tilav/sw/blob/2019061/projects/2019061/README.md).

**Comment by StavroulaAnatolaki** [comment]
το command prompt πρεπει να είναι ο ΑΜ

**Reply by tilav** [reply]
Σε ευχαριστώ για την διευκρίνιση! Το γνωρίζω, αλλά θα κάνω το παραδοτέο με άλλο λειτουργικό όταν το εγκαταστήσω.
